### PR TITLE
deployment section

### PR DESCRIPTION
**Pull request by joshuafernandes**
## Pull Request Description
New upto date deployment section for the various tooling we provide

### CODE REVIEW

File: docs/Deployment/Ansible.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+with other orchestration tooling like Terraform.
+
+## Ansible-Galaxy
+
+You can use an [Ansible Galaxy](https://galaxy.ansible.com/consensys/goquorum) role which will install, configure and run
+GoQuourum
+
+### Install the role
+
+The first step to include the role in a `requirements.yaml` with any other roles that you use. For example the prometheus
```
Comment (NicolasMassart) [inline docs/Deployment/Ansible.md:15]:
```suggestion
The first step to include the role in a `requirements.yaml` with any other roles that you use. For example the Prometheus
```

File: docs/Deployment/Ansible.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+exporter roles below will publish system metrics that can be scraped with prometheus
```
Comment (NicolasMassart) [inline docs/Deployment/Ansible.md:16]:
```suggestion
exporter roles below will publish system metrics that can be scraped with Prometheus
```

File: docs/Deployment/Ansible.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+      goquorum_consensus_algorithm: "istanbul"
+      goquorum_genesis_path: "/path/to/genesis_file"
+      goquorum_http_enabled: true
+      goquorum_http_host: 127.0.0.1
+      goquorum_http_port: 8545
+      goquorum_http_api: ["admin","db","eth","debug","miner","net","web3","quorum","ibft"]
+      goquorum_http_cors_origins: ["all"]
+      goquorum_http_virtual_hosts: ["all"]
+
+```
```
Comment (bgravenorst) [inline docs/Deployment/Ansible.md:62]: @joshuafernandes is there a missing step here? Does the user need to run Ansible after defining the variables here?

Comment (joshuafernandes) [reply docs/Deployment/Ansible.md:62]: Yep :100: thankyou @bgravenorst , have just pushed a fix for this

**Review by bgravenorst — APPROVED** [review]
Edited the content. LGTM.

**Pull request merged**